source/Engine.cpp

```diff
@@ -1192,12 +1192,16 @@ list<ShipEvent> &Engine::Events()
 // Draw a frame.
 void Engine::Draw() const
 {
-	Point motionBlur = Preferences::Has("Render motion blur") ? centerVelocity : Point();
+	Point motionBlur = centerVelocity;
+	int baseBlur = Preferences::Has("Render motion blur") ? 1 : 0;
 
 	Preferences::ExtendedJumpEffects jumpEffectState = Preferences::GetExtendedJumpEffects();
+
```

Comment:
It's going to be converted to a double during calculations anyway.
```suggestion
	double baseBlur = Preferences::Has("Render motion blur") ? 1. : 0.;

	Preferences::ExtendedJumpEffects jumpEffectState = Preferences::GetExtendedJumpEffects();
```